WordPress/gutenberg
# Query blocks - Support wide alignments in editor.

## Description

EDIT - Updated https://github.com/WordPress/gutenberg/pull/24229#issuecomment-665328780

Currently in the site editor, the Query Loop, Query, and Post Content blocks are given a default wrapper that causes their width to be restricted.  A post may have a block with a wide of full block alignment setting.  The post editor, the front end for the post, and the homepage (if set to render current posts) will render the content at full width:
![Screen Shot 2020-07-27 at 5 58 02 PM](https://user-images.githubusercontent.com/28742426/88597082-76e7c900-d034-11ea-8394-f258cf009883.png)

While in the site editor the width is restricted by the Query Loop and Post Content blocks:
![Screen Shot 2020-07-27 at 5 58 26 PM](https://user-images.githubusercontent.com/28742426/88597269-e1990480-d034-11ea-880f-4cfa747b4db3.png)

If we add styles to set the max-width for these blocks inside the editor, or allow for supporting alignment in the Query and Query Loop blocks, they can be less restrictive and in the editor we would have:
![Screen Shot 2020-07-27 at 6 06 47 PM](https://user-images.githubusercontent.com/28742426/88597813-178ab880-d036-11ea-90e4-07fed871632c.png)


## How has this been tested?



Tested on local docker environment.

* Create a post with some wide/full alignment blocks (such as cover block).
* Load a template that will render this post within the Query or QueryLoop blocks (such as an index template for showing recent posts as homepage).
* Verify that in the editor the wide/full alignment blocks are not restricted in width by the parent query and post content blocks.

## Screenshots 

## Types of changes





## Checklist:
